### ISSUE TITLE
 How do I organize my data to evaluate unlabeled test data in pix2pix?

### ISSUE THREAD

**Issue by kimtaehyeong** [junyanz/pytorch-CycleGAN-and-pix2pix#932]
Hello,
How do I organize my data to test.py unlabeled test data in pix2pix?
I configured it like a 'facades' dataset for learning.
But there is no label for test data. Should I test it by creating a label on an empty image like the train, val data configuration?
Thank you.

**Comment by junyanz** [comment]
See [README](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix#apply-a-pre-trained-model-cyclegan). You can use the `--model test`. See an example script `./scripts/test_single.sh`.

**Comment by kimtaehyeong** [comment]
thank you!
